api/resolvers/wallet.js

```diff
@@ -506,6 +527,15 @@ const resolvers = {
       }
       return { id }
     },
+    setWalletPriority: async (parent, { id, priority }, { me, models }) => {
```

Comment:
setting wallet priority is now an isolated mutation so that we don't have to revalidate everything.